Xwt.WPF/Xwt.WPFBackend/TreeViewBackend.cs

```
@@ -234,6 +235,59 @@ public void UnselectAll ()
+			((TreeStoreBackend)sourceBackend).NodeChanged += TreeViewBackend_NodeChanged;
+		}
+
+		private void TreeViewBackend_NodeChanged (object sender, TreeNodeEventArgs e)
+		{
+			try {
+				var treeView = (TreeView)Frontend;
+				var columns = treeView.Columns;
+
+				if (!cellViews.Any ())
+					return;
+
+				var changedCell = cellViews.Where (cv => cv.Value.GlobalCellIndex == e.ChangedCellIndex)
+					.Select (cv => (KeyValuePair<CellView, CellInfo>?)cv)
+					.FirstOrDefault ();
+				if (changedCell == null)
+					return;
+
+				var changedColumnIndex = changedCell.Value.Value.ColumnIndex;
+				//Debug.WriteLine("changedColumnIndex:" + changedColumnIndex + " e.ChangedCellIndex: " + e.ChangedCellIndex + " columns.Count: " + columns.Count);
+
+				if (changedColumnIndex < 0 || changedColumnIndex > columns.Count - 1)
+					return;
+
+				var column = columns [changedColumnIndex];
+				var treeViewItem = GetVisibleTreeItem (e.Node);
+				if (treeViewItem == null || !column.Expands || !column.Views.Any ())
+					return;
+
+				double defaultColumnSpacing = 6;
+				var row_bounds = treeView.GetRowBounds (e.Node, false);
+
+				double totalColumnWidth = 0;
+				if (changedColumnIndex == columns.Count - 1) {
+					var cell_bg_bounds = treeView.GetCellBounds (e.Node, column.Views [0], true);
+					totalColumnWidth = row_bounds.Right - cell_bg_bounds.Left + 2 * defaultColumnSpacing;
+				} else {
+					totalColumnWidth = 2 * defaultColumnSpacing;
+					double maxColumnViewWidth = 0;
+					foreach (var cell in column.Views) {
+						var cell_bounds = treeView.GetCellBounds (e.Node, cell, false);
+
+						if (cell_bounds.Width > maxColumnViewWidth)
+							maxColumnViewWidth = cell_bounds.Width;
+					}
+
+					totalColumnWidth += maxColumnViewWidth;
+				}
+
+				treeViewItem.ExpandColumnWidth (changedColumnIndex, totalColumnWidth);
+			} catch (Exception ex) {
+				Console.WriteLine ($"[TreeViewBackend_NodeChanged] Exception: {ex}");

```

Is there a reason to swallow the exception here?